python/cugraph/cugraph/dask/comms/comms.py

```diff
@@ -81,8 +81,8 @@ def subcomm_init(prows, pcols, partition_type):
     __subcomm = (prows, pcols, partition_type)
 
 
-def _subcomm_init(sID, partition_row_size):
-    handle = get_handle(sID)
+def _subcomm_init(sID, partition_row_size, dask_worker):
```

Comment:
```suggestion
def _subcomm_init(sID, partition_row_size, dask_worker=None):
```

Reply:
Minor suggestion, not generally a big problem unless you call it outside of `client.run` as well, causing `dask_worker` not to be automatically populated and thus force you to pass `dask_worker` as a positional argument.

Reply:
Got it. I just fixed it. Thanks